service/dap/utils.go

```diff
@@ -0,0 +1,12 @@
+package dap
+
+type tuple struct {
```

Comment:
This is an unusual data structure for Go, I'm not sure why you need it and there are no comments in this file

Reply:
It's gone.